ci-operator/config/openstack-k8s-operators/barbican-operator/openstack-k8s-operators-barbican-operator-main.yaml

```diff
@@ -1,5 +1,52 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  openstack-operator-ci-tempest:
+    name: tools
+    namespace: openstack-k8s-operators
+    tag: openstack-operator-tempest-container
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: latest
 build_root:
   from_repository: true
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install python-openstackclient
+    COPY oc /usr/bin/oc
+    COPY opm /usr/bin/opm
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    opm-builder:
+      paths:
+      - destination_dir: .
+        source_path: /bin/opm
+  to: openstack-operator-ci-build-deploy
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install pre-commit
+  from: src
+  to: openstack-operator-ci-pre-commit
+- build_args:
+  - name: KUTTL_VERSION
+    value: v0.15.0
+  dockerfile_literal: |
```

Comment:
If you are not going to add kuttl test here, you should remove this image build from here also, since is not being used.